Accessories/Kaman_IO_ConverterBoard/REV20230213C/io-converter-board-bom.csv

```diff
@@ -0,0 +1,18 @@
+Quantity,Supplier Part Number 1,Designator,Comment,Value,Footprint
```

Comment:
Let's revise our columns to match the [AMDS REV20200920D/mainboard-bom.csv](https://github.com/Severson-Group/AMDS/blob/develop/Mainboard/REV20200920D/mainboard-bom.csv). 

At least make the first several columns be identical. Maybe it makes sense to still keep the value and footprint fields.